hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneClientFactory.java

```diff
@@ -136,6 +136,31 @@ public static OzoneClient getRpcClient(String omHost, Integer omRpcPort,
     return getRpcClient(config);
   }
 
+  /**
+   * Returns an OzoneClient which will use RPC protocol.
+   *
+   * @param omServiceId
+   *        Service ID of OzoneManager HA cluster.
+   *
+   * @param config
+   *        Configuration to be used for OzoneClient creation
+   *
+   * @return OzoneClient
+   *
+   * @throws IOException
+   */
+  public static OzoneClient getRpcClient(String omServiceId,
```

Comment:
One suggestion, can we have a single getRpcClient method, instead of multiple methods. Because in all cases, I think we can call this method if the configuration is set with om address in non-HA case.

Reply:
@bharatviswa504 Yeah I believe we discussed about that and we agreed to do this. But merging all those `getRpcClient()` calls will change something out of the scope of this jira. We should probably open another jira to merge those functions.

Reply:
Filed https://issues.apache.org/jira/browse/HDDS-2105